build-scripts/hack/util.py

```diff
@@ -48,3 +49,26 @@ def read_file(path: Path) -> str:
 
 def read_url(url: str) -> str:
     return urlopen(url).read().decode().strip()
+
+
+def helm_pull(chart_name: str, repo_url: str, version: str, destination: Path) -> None:
+    parse_output(
+        [
+            "helm",
+            "pull",
+            f"{repo_url}",
+            f"{chart_name}/{chart_name}",
```

Comment:
Please update this command, this is not the right invocation. It might work locally because you already have the repo, but it is not right. See:

```
$ helm pull https://metallb.github.io/metallb metallb/metallb --version 0.14.5
Error: repo metallb not found
```

The correct way to write this command is

```
$ helm pull metallb --repo https://metallb.github.io/metallb --version 0.14.5
```

Reply:
this is not addressed